### PR TITLE
Skeleton Grabs Enemies Until One Dies If Ordered Or It Gets In Range

### PR DESCRIPTION
**Pull request by Anisetobegan**

### CODE REVIEW

File: Assets/Scripts/Enemies.cs [hunk 1]
```diff
@@ -37,4 +37,14 @@ virtual protected void Die()
         Destroy(gameObject);
     }
 
+    virtual public void IsBeingGrabbed()
```
Comment (morathias) [inline Assets/Scripts/Enemies.cs]: Add a bool parameter and set it to isStopped

File: Assets/Scripts/Enemies.cs [hunk 2]
```diff
@@ -37,4 +37,14 @@ virtual protected void Die()
...
+    {
+        agent.isStopped = true;
+    }
+
+    virtual public void IsNotBeingGrabbed()
+    {
```
Comment (morathias) [inline Assets/Scripts/Enemies.cs]: remove this

**Review by morathias — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Enemies.cs (hunk 1 above)
Comment (Anisetobegan) [reply Assets/Scripts/Enemies.cs]: Done

File: Assets/Scripts/Enemies.cs (hunk 2 above)
Comment (Anisetobegan) [reply Assets/Scripts/Enemies.cs]: Done

**Pull request merged**